### PR TITLE
create jenkins agent with packer pre-installed

### PR DESCRIPTION
**Pull request by staskolauren**

### CODE REVIEW

**Comment by bpholt** [comment]
I see a lot of warnings in the actions output about GHA functionality that will soon be removed. Do we need to update the upstream workflow?

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+    tags: [v*]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+      - name: Build and Push Docker Image
+        uses: Dwolla/jenkins-agents-workflow/.github/actions/build@main
```
Comment (bpholt) [inline .github/workflows/ci.yml:17]: I wonder if we should be using a more explicit version than just `@main`—could tie into updating the workflow to resolve all the warnings I mentioned.

**Review by bpholt — APPROVED** [review]

**Comment by staskolauren** [comment]
> I see a lot of warnings in the actions output about GHA functionality that will soon be removed. Do we need to update the upstream workflow?

Ooh good point! Yeah looking through the [raw logs](https://pipelines.actions.githubusercontent.com/serviceHosts/c42a517b-7ac5-4925-bf49-2c1aff705787/_apis/pipelines/1/runs/2/signedlogcontent/2?urlExpires=2023-08-25T13%3A46%3A37.2998039Z&urlSigningMethod=HMACV1&urlSignature=IvPjPAYhHKdjJjJf4r3m%2FEjKd3LWK4Hu7TBuxClJ4hA%3D) the deprecated syntax seems to be in all of the downstream common actions (minus docker-login) that [our dwolla common action](https://github.com/Dwolla/jenkins-agents-workflow/blob/main/.github/actions/build/action.yml) uses 😬

**Comment by staskolauren** [comment]
> > I see a lot of warnings in the actions output about GHA functionality that will soon be removed. Do we need to update the upstream workflow?
> 
> Ooh good point! Yeah looking through the [raw logs](https://pipelines.actions.githubusercontent.com/serviceHosts/c42a517b-7ac5-4925-bf49-2c1aff705787/_apis/pipelines/1/runs/2/signedlogcontent/2?urlExpires=2023-08-25T13%3A46%3A37.2998039Z&urlSigningMethod=HMACV1&urlSignature=IvPjPAYhHKdjJjJf4r3m%2FEjKd3LWK4Hu7TBuxClJ4hA%3D) the deprecated syntax seems to be in all of the downstream common actions (minus docker-login) that [our dwolla common action](https://github.com/Dwolla/jenkins-agents-workflow/blob/main/.github/actions/build/action.yml) uses 😬

Oh - interestingly it [looks like](https://github.blog/changelog/2023-07-24-github-actions-update-on-save-state-and-set-output-commands/) github decided to continue supporting these, but still print warnings about them 🤷‍♀️

**Pull request merged**